src/platforms/ios/iosutils.mm

```diff
@@ -61,3 +62,9 @@
   NSString* encodedReceipt = [receipt base64EncodedStringWithOptions:0];
   return QString::fromNSString(encodedReceipt);
 }
+
+void IOSUtils::appReviewRequested() {
+  if ([SKStoreReviewController class]) {
+    [SKStoreReviewController requestReview];
```

Comment:
Apple does not recommend using `requestReview` on button presses as it does not always open a popup [(requestReview docs)](https://developer.apple.com/documentation/storekit/skstorereviewcontroller/2851536-requestreview)

We should rather open a link that leads to the app store like outlined [here](https://developer.apple.com/documentation/storekit/requesting_app_store_reviews) under `Manually Request a Review`

Reply:
Good point! We should rise up this issue with the UX team. Follow up?